packages/Php/tests/Rector/BinaryOp/IsCountableRector/Fixture/fixture71.php.inc

```diff
@@ -0,0 +1,17 @@
+<?php
+
+function isCountable()
+{
+    is_array($foo) || $foo instanceof Countable;
+}
+
+?>
+-----
+<?php
+
+function isCountable()
+{
+    is_array($foo) || $foo instanceof Countable;
+}
+
+?>
```

Comment:
If code are the same before/after (= no change), you can remove the part under `-----` (included)

```diff
 <?php
 function isCountable()
 {
     is_array($foo) || $foo instanceof Countable;
 }
 ?>
------
-<?php
-function isCountable()
-{
-    is_array($foo) || $foo instanceof Countable;
-}
-?>
```